llarp/constants/limits.cpp

```diff
@@ -0,0 +1,13 @@
+#include <constants/limits.hpp>
+
+namespace llarp
+{
+  namespace limits
+  {
+    /// snode limit parameters
+    const LimitParameters snode = {6, 60};
```

Comment:
is this something we would want to override at the command line/environment at all?

Reply:
yes and we do